docs/src/content/docs/utilities/host-binding.md

```diff
@@ -0,0 +1,46 @@
+---
+title: hostBinding
+description: ngxtension/host-binding
+---
+
+`hostBinding` is a function that returns either a _writable_ or _readonly_ signal and binds the value held in that signal to the host property passed as the first argument like `@HostBinding` would do.
+
+```ts
+import { hostBinding } from 'ngxtension/host-binding';
+```
+
+## Usage
+
+With `@HostBinding` you can bind a color from a class property:
+
+```ts
+@Component({
+  standalone: true;
+  selector: 'my-component'
+  template: '...'
+})
+export class MyComponent {
+  @HostBinding('style.color') color = 'red';
+
+  updateColor(color: 'red' | 'blue') {
+    this.color = color;
+  }
+}
+```
+
+With `hostBinding` you can now bind anything like `@HostBinding` on writable or readonly signals:
+
+```ts
+@Component({
+  standalone: true;
+  selector: 'my-component'
+  template: '...'
+})
+export class MyComponent {
+  color = hostBinding('style.color', signal('red'));
+
+  updateColor(color: 'red' | 'blue') {
+    this.color.set(color);
+  }
```

Comment:
question: can we add a usage in combination with Input? There will be 2 cases:
- Observable Input like people do today with setter Inputs

```ts
export class MyComponent {
    #color = new BehaviorSubject('red');
    @Input() set color(color: 'red' | 'blue') {
        this.#color.next(color);
    }    
    // how to use hostBinding() in this case
}
```

- Future _theoretical_ signal inputs

```ts
export class MyComponent {
    color = input('red');
    constructor() {
        // maybe this is how we'd use this.
        hostBinding('style.color', this.color);
    }
}
```

Without Signal Input, can we also add a usage with Input?

```ts
export class MyComponent {
    #color = hostBinding('style.color', signal('red'));
    
    @Input() set color(color: 'red' | 'blue') {
        this.#color.set(color);
    }
}
```

Reply:
About observables I thought about maybe creating a custom operator instead, but I'm not sure! What do you think?

I don't really know how signal based input will work but `hostBinding` could work the exact same way that of any `Signal`

About your last use case, it will work exactly as you wrote it 😁